babel.config.js

```diff
@@ -114,6 +114,18 @@ module.exports = function getBabelConfig(api) {
           ],
         ],
       },
+      stable: {
```

Comment:
@oliviertassinari this change is breaking the `yarn test:karma` scenario.

Reply:
I remember that we added it to support the imports like `import SliderUnstyled from '@material-ui/unstyled/SliderUnstyled`, but now it fails the test scenario.